packages/mermaid/src/docs/intro/index.md

```diff
@@ -1,34 +1,37 @@
 
 <!-- Mermaid book banner -->
 
 [![Explore Mermaid.js in depth, with real-world examples, tips & tricks from the creator... The first official book on Mermaid is available for purchase. Check it out!](img/book-banner-post-release.jpg)](https://mermaid-js.github.io/mermaid/landing/)
 
+## Why use Mermaid
+
 <!-- <Main description> -->
 
-Mermaid is a JavaScript based diagramming and charting tool that uses Markdown-inspired text definitions and a renderer to create and modify complex diagrams. The main purpose of Mermaid is to help documentation catch up with development.
+- Create concepts
+
+  Anyone who has ever used Visio, or Excel knows how hard it is to create, edit and maintain good visualizations with those tools. Mermaid offers a quicker, less complicated, and more convenient way of going from concept to visualization.
+
+- Keep the documentation up with the development
+
+  Doc-Rot kills diagrams as quickly as it does text, but it takes hours in a desktop application to produce a diagram. Mermaid addresses this problem by providing users with text-based easily modifiable diagrams. It can also be embedded in the development pipeline and used by scripts.
+
+- Diagram as Code
 
-> Doc-Rot is a Catch-22 that Mermaid helps to solve.
+  One definition leads to one layout, so your code becomes idempotent. It guarantees consistency. It becomes far easier to see what changes was made and possible to keep diagram in version control system along with code.
 
-Diagramming and documentation costs precious developer time and gets outdated quickly.
-But not having diagrams or docs ruins productivity and hurts organizational learning.<br/>
-Mermaid addresses this problem by enabling users to create easily modifiable diagrams, it can also be made part of production scripts (and other pieces of code).<br/>
-<br/>
-Mermaid allows even non-programmers to easily create detailed and diagrams through the [Mermaid Live Editor](https://mermaid.live/).<br/>
-[Tutorials](../config/Tutorials.md) has video tutorials.
-Use Mermaid with your favorite applications, check out the list of [Integrations and Usages of Mermaid](../ecosystem/integrations.md).
+Mermaid allows even non-programmers to easily create detailed and diagrams through the [Mermaid Live Editor](https://mermaid.live/).
+There are plenty of [Tutorials](../config/Tutorials.md) made by our woderful community. Also, Mermaid has a huge amount of [Use-cases and Integrations](../ecosystem/integrations.md), so you can use it with your favorite applications.
 
-For a more detailed introduction to Mermaid and some of its more basic uses, look to the [Beginner's Guide](../community/n00b-overview.md) and [Usage](../config/usage.md).
+For a more detailed introduction to Mermaid and some of its more basic uses, look to the [Beginner's Guide](../intro/n00b-gettingStarted.md) and [Usage](../config/usage.md).
```

Comment:
Can we remove the `n00b` from the file names?

We should setup the redirects in redirects.ts file.

Reply:
I decided not to redo everything in one batch, that will be done in upcoming requests. In case of applied changes to a file and its simultaneous renaming diff may become meaningless.